### PR TITLE
New build system

### PR DESCRIPTION
**Pull request by matiboy**
- move to a better build system which checks AoT
- some linting fixes

This would be a version 3.0.0

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
-.DS_Store
-# Ignored files
-*.js
-*.map
-*.d.ts
-**/*.ngfactory.ts
-**/*.metadata.json
-**/*.ngsummary.json
-.vscode
+$ cat .gitignore
```
Comment (pfeigl) [inline .gitignore]: This seems like a copy paste from the bash which shouldn't be part of the gitignore

File: package.json [hunk 2]
```diff
@@ -1,51 +1,78 @@
...
+    "author": "",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/matiboy/angular2-prettyjson.git"
+    },
+    "bugs": {
+        "url": "https://github.com/matiboy/angular2-prettyjson/issues"
+    },
+    "homepage": "https://github.com/matiboy/angular2-prettyjson",
+    "keywords": [
```
Comment (pfeigl) [inline package.json:27]: I'd keep json and pretty within the keywords

**Review by pfeigl — COMMENTED** [review]
Added two small comments and also realized that the few existing unit tests are gone now - is this by intention?

Other than that, LGTM.

Also tried angular2-prettyjson@next in our project and everything seems to work just fine

**Comment by matiboy** [comment]
@pfeigl You're right about the missing unit tests. Re-added them.
Going to publish now

**Pull request merged**